src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DefaultSize = 10;
+    private static final String DefaultColor = "White";
+    private static final int Limit = 100;
+    private static final int TotalFigure = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int indexFigure = random.nextInt(TotalFigure);
+        switch (indexFigure) {
+            case 1:
+                return new Square(colorSupplier.getRandomColor(),
+                        random.nextInt(Limit));
```

Comment:
lets make private methods to get each figure - this will make your code easier to read
my last comment wasn't sent))
```suggestion
            case 1:
                return getRandomSquare();
```